src/components/hero.tsx

```diff
@@ -0,0 +1,65 @@
+'use client'
+import React, { useState, useEffect } from 'react';
+import OneWayComponent from './OneWayComponent';
+import RoundTripComponent from './Roundtrip';
+
+const Hero = () => {
+  // State to manage which component to display
+  const [displayComponent, setDisplayComponent] = useState('oneWay');
+
+  const handleComponentChange = (component:any) => {
+    setDisplayComponent(component);
+  };
```

Comment:
`handleComponentChange` function isn't being used anywhere. Kindly always remove the unused code before pushing it to make the codebase cleaner. 👍